src/status_im/navigation/effects.cljs

```diff
@@ -74,9 +74,7 @@
    (let [[status-bar-theme] (get-status-nav-color root-id theme)
          root               (get (roots/roots status-bar-theme) root-id)]
      (dismiss-all-modals)
-     (rf/dispatch [:profile.settings/switch-theme
-                   (get roots/themes root-id)
```

Comment:
[Issue](https://github.com/status-im/status-mobile/issues/20980): Here: shell-stack was using nil value for the theme, which led to fallback of dark and created conflict